.gitignore

```diff
@@ -26,3 +26,6 @@ yalc.lock
 coverage*
 reports*
 2
+
+# Diagnostic reports (https://nodejs.org/api/report.html)
+report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json
```

Comment:
Could this just be `report.*.json`?

Reply:
I used this because it was the 'official' line from gitignore.io. Don't have a strong opinion either way if you think that is better.

Reply:
```suggestion
report.*.json
```